lib/utils/Utils.dart

```diff
@@ -1128,10 +1139,10 @@ class Utils {
     }
   }
 
-  static getLanguage() {
-    if (ui.window != null && ui.window.locale != null) {
-      String stringLanguange = ui.window.locale.languageCode;
-      String? stringCountry = ui.window.locale.countryCode;
+  static ui.Locale getLanguage() {
```

Comment:
not sure why `ui.Locale` added here. @DigitalSeneca any opinion?

Reply:
It's actually returning a Locale, both of those options would be correct I think

```
static Locale getLanguage() {
static ui.Locale getLanguage() {
```